cypress/e2e/tasks.test.ts

```diff
@@ -242,30 +242,22 @@ http.post(
 
     it('can delete a task', () => {
       cy.getByTestID('task-card')
-        .first()
-        .trigger('mouseover')
+        // .first()
```

Comment:
I don't entirely understand why removing `.first()` makes this work except for there being only 1 card and it's not an array, so trying to grab the first item doesn't actually make sense